test/index.js

```diff
@@ -96,3 +96,24 @@ test('check error throwing', async (t) => {
 
   t.end()
 })
+
+test('.byArguments() > should call function concurrently when arguments are equal', async (t) => {
```

Comment:
concurrently = at the same time, so it should NOT run concurrently when they're equal
```suggestion
test('.byArguments() > should call function concurrently only when arguments change', async (t) => {
```

Reply:
```suggestion
test('.byArguments() > should limit concurrency when arguments are equal', async (t) => {
```
???

Reply:
@RyanZim lgtm!